src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,97 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] elements;
```

Comment:
It is better to use a parameterized type
```suggestion
    private T [] elements;
```